src/defaults/definitions.js

```diff
@@ -38,5 +38,25 @@ module.exports = {
     period: 60,
     evaluationPeriods: 1,
     comparisonOperator: 'GreaterThanThreshold',
-  }
-};
\ No newline at end of file
+  },
+  bunyanWarnings: {
```

Comment:
OK, makes sense. Do we have a plan to submit PR to the original repo? Or we are planning to create our own version? In any case, documentation should be updated.